cmd/cloud.go

```diff
@@ -305,10 +327,75 @@ func cloudCmdFlagSet() *pflag.FlagSet {
 	// K6_EXIT_ON_RUNNING=true won't affect the usage message
 	flags.Lookup("exit-on-running").DefValue = "false"
 
+	// read the comments above for explanation why this is done this way and what are the problems
+	flags.BoolVar(&disableTailLogs, "disable-tailing-logs", disableTailLogs,
+		"disable the automatic tailing of logs when a test is executed in the cloud")
+	flags.Lookup("disable-tailing-logs").DefValue = "false"
+
+	return flags
+}
+
+//nolint:gochecknoglobals
+var cloudLogsCmd = &cobra.Command{
```

Comment:
Given the issues we've had, why not delay this the command to get old logs for k6 v0.29.0? As far as I understand, it's not going to work very well now, right? And it's far from essential, especially when we still have `k6cloudlogs` doing essentially the same thing? So, personally, I'd vote for not including this now.

Reply:
This is a possibility, but likely I will be able to make the changes ... today or tomorrow. 
But I agree that maybe it should be in a separate PR ...

Reply:
I don't doubt we can make the changes, but why not wait a version for things to be a bit more stable? There's no rush to have this command, the other use case in this PR is much more important...